explanation/security/charms.md

```diff
@@ -27,3 +27,33 @@ When Anbox Cloud is deployed without the use of an external CA, the charms will
 ### Packages used
 
 * [cryptography from PyPI](https://pypi.org/project/cryptography/)
+
+## Juju secrets
+
+Confidential data such as private keys, API tokens, or credentials managed by the charm or passed through relations between charms is securely handled using [Juju secrets](https://documentation.ubuntu.com/juju/3.6/reference/secret/) if you deploy Anbox Cloud with Juju 3.0 or later. You can use the following command to view the secrets used by charms:
+
+    ```
+    juju list-secrets
+    ```
+
+Currently, all Juju secrets are managed by the charms themselves, not by the model. This means you can view them, but you cannot rotate or remove them.
+```{note}
+Juju secrets are disabled in Anbox Cloud deployments based on Juju 2.9, as the secret feature is only available starting from Juju 3.0.
+```
+
+### Use Juju secrets across model
+
+When using Juju secrets to protect data shared across units in different models, you need to establish a cross-model relation. For example, the `nats` charm provides messaging services, and the `anbox-stream-agent` charm acts as a NATS client. If `anbox-stream-agent` and `nats` are deployed in different models, follow the steps below:
+
+1. Switch to the model containing `nats` and create an offer:
+
+    juju switch <model containing nats>
+    juju offer nats:client
```

Comment:
This doesn't render correctly, I'm not sure what's the correct fix.
Please take a look at the render on https://github.com/canonical/anbox-cloud-docs/blob/53d316638376029ee23bbe1bae216897d0b0b1d6/explanation/security/charms.md

Reply:
Just fixed.